app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -606,7 +631,7 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener, Camera
             toggle_flash.beVisible()
         } else {
             toggle_flash.beInvisible()
-            toggle_flash.setImageResource(R.drawable.ic_flash_off_vector)
+            toggle_flash.icon = AppCompatResources.getDrawable(this, R.drawable.ic_flash_off_vector)
```

Comment:
any specific reason for using compat here?

Reply:
No specific reason. Just an old habit. 
Since we have `minSDK 29`, we can just call getDrawable but then we have the warning on Android Studio. 
Do you want me to change this?